chromium_src/ios/web/webui/url_data_manager_ios_backend.mm

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "ios/web/webui/url_data_manager_ios_backend.h"
+
+#define ShouldDenyXFrameOptions ShouldDenyXFrameOptions());   \
+  job->set_add_content_security_policy(                       \
+      source->source()->ShouldAddContentSecurityPolicy());    \
+  job->set_content_security_policy_object_source(             \
```

Comment:
this is already called in `URLDataManagerIOSBackend::StartRequest`, why are we calling it again?